src/configure/resources/constants.ts

```diff
@@ -131,4 +131,15 @@ export const ReservedHostNames: string[] = [
     "teamservices",
     "java",
     "beta"
-];
\ No newline at end of file
+];
+
+export const SelectFolderOrRepository = 'selectFolderOrRepository';
```

Comment:
I believe these are being used i telemetry. Do you want to keep these constants in telemetryKeys file and rename that to telemetryConstants just so we have telemetry related strings together ?

Reply:
Did not keep them there as these are not the keys rather the value. They key is would be 'CurrentInput' and these would be the value. hence have kept them in constants which are related to the input